### ISSUE TITLE
Meaning of Grey Entries in DNS Log

### ISSUE THREAD

**Issue by rhclayto** [Gedsh/InviZible#74]
In the documentation for the real-time DNS log, it explains the meaning of various log entries:

*Red: DNS query blocked or no response
*Yellow: Application connects to IP address using DNS cache or does not use plaintext DNS at all
*Green: DNS resolved successfully.

But I also see some grey log entries, which look like either of the following:

`forecast.weather.gov -> sancert.weather.gov.edgekey.net`

or

`example.com -> 10.100.123.12`

In other words, domain name -> domain name, or domain name -> IP address. onion addresses also seem to exhibit the second pattern.

What do the grey entries mean?

**Comment by Gedsh** [comment]
Gray entries mean DNS was resolved, but no application was using the returned IP. I need to update the manuals.

**Comment by Gedsh** [comment]
https://github.com/Gedsh/InviZible/wiki/Real-time-connections-log
